src/Servers/Kestrel/Kestrel/test/KestrelConfigurationLoaderTests.cs

```diff
@@ -922,6 +922,92 @@ public async Task CertificateChangedOnDisk(bool reloadOnChange)
         }
     }
 
+    [Fact]
+    [OSSkipCondition(OperatingSystems.Windows)] // Windows has poor support for directory symlinks (e.g. https://github.com/dotnet/runtime/issues/27826)
```

Comment:
It ran and failed on Windows anyway?
https://github.com/dotnet/aspnetcore/pull/50251/checks?check_run_id=16088670360

Reply:
`ConditionalFact`?

Reply:
Ah.